core/commands/slash/commit.ts

```diff
@@ -4,8 +4,8 @@ import { stripImages } from "../../llm/images.js";
 const CommitMessageCommand: SlashCommand = {
   name: "commit",
   description: "Generate a commit message for current changes",
-  run: async function* ({ ide, llm, input }) {
-    const diff = await ide.getDiff();
+  run: async function* ({ ide, llm, params }) {
+    const diff = await ide.getDiff(!params?.includeUnstaged);
```

Comment:
I think something like this would be cleaner, where we define `INCLUDE_UNSTAGED_DEFAULT = false` as a class property.
```suggestion
    const includeUnstaged = params ? params.includeUnstaged : INCLUDE_UNSTAGED_DEFAULT
    const diff = await ide.getDiff(includeUnstaged);
```